src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -494,3 +501,18 @@ void pychip_Stack_SetLogFunct(LogMessageFunct logFunct)
     // Ideally log redirection should work so that python code can do things
     // like using the log module.
 }
+
+CHIP_ERROR pychip_DeviceController_PostTaskOnChipThread(void * pythonContext)
```

Comment:
I think if you merged those two functions into a single one that takes both the callback and the context, it could be reused for other purposes. By using the global variable you limit the usage to a single callback.

Reply:
i think it has been resolved.